pandas/core/indexes/base.py

```diff
@@ -264,7 +264,7 @@ def __new__(cls, data=None, dtype=None, copy=False, name=None,
             if fastpath:
                 return cls._simple_new(data, name)
 
-        from .range import RangeIndex
+        from pandas import RangeIndex
```

Comment:
Can you convert these to absolute references rather than pulling from the top level?

Reply:
this is ok, we do this all over the place